tests/ffi/dev-mem_not_present/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+check_dev-mem_not_present(){
+    # Check /dev/mem is not present in QM partition
+    if podman exec qm test -e /dev/mem &>/dev/null; then
```

Comment:
Nitpicky: you don't need the output redirection, as `test` will only return something

Reply:
That comment aside it looks good to me

Reply:
Sure, done.